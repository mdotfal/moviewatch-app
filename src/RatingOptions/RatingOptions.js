import React, { Component } from 'react';

class RatingOptions extends Component {
  render() {
    return(
      <div className="content-rating">
        <fieldset>
          <legend>Choose one:</legend>
          <label htmlFor="rating-watch">
            <input type="radio" id="rating-watch" name="rating" value="Watch" required /> Watch
          </label>
          <label htmlFor="rating-skip">
            <input type="radio" id="rating-skip" name="rating" value="Skip" required /> Skip
          </label>
        </fieldset>
      </div>
    );
  }
};

export default RatingOptions;