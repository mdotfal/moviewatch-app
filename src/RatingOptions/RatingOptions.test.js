import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RatingOptions from './RatingOptions';

describe( `RatingOptions component`, () => {
  it(Â `renders without errors`, () => {
    const wrapper = shallow( <RatingOptions /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })
})