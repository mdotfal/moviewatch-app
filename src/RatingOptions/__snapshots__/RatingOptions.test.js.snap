// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingOptions component renders without errors 1`] = `
<div
  className="content-rating"
>
  <fieldset>
    <legend>
      Choose one:
    </legend>
    <label
      htmlFor="rating-watch"
    >
      <input
        id="rating-watch"
        name="rating"
        required={true}
        type="radio"
        value="Watch"
      />
       Watch
    </label>
    <label
      htmlFor="rating-skip"
    >
      <input
        id="rating-skip"
        name="rating"
        required={true}
        type="radio"
        value="Skip"
      />
       Skip
    </label>
  </fieldset>
</div>
`;
